Compilers.sln

```diff
@@ -159,7 +158,6 @@ Global
 		src\Compilers\Core\CommandLine\CommandLine.projitems*{4b45ca0c-03a0-400f-b454-3d4bcb16af38}*SharedItemsImports = 4
 		src\Dependencies\PooledObjects\Microsoft.CodeAnalysis.PooledObjects.projitems*{5f8d2414-064a-4b3a-9b42-8e2a04246be5}*SharedItemsImports = 4
 		src\Compilers\Core\CommandLine\CommandLine.projitems*{7ad4fe65-9a30-41a6-8004-aa8f89bcb7f3}*SharedItemsImports = 4
-		src\Compilers\Core\CommandLine\CommandLine.projitems*{8ce3a581-2969-4864-a803-013e9d977c3a}*SharedItemsImports = 4
```

Comment:
What are those changes?

Reply:
Ah, did not intend to check this in, let me revert.

Reply:
I may have screwed this up in https://github.com/dotnet/roslyn/pull/22142/files ... actually yes, the `SharedItemsImports` deletion is correct, as the two GUIDs deleted in this PR are CscCore and VbcCore.

I'm not sure about the `VisualStudioVersion` change or the `GlobalSection(ExtensibilityGlobals)` addition.